docs/sources/alerting/notifications.md

```diff
@@ -75,20 +75,22 @@ able to access the image.
 
 {{< imgbox max-width="40%" img="/img/docs/v4/slack_notification.png" caption="Alerting Slack Notification" >}}
 
-To set up slack you need to configure an incoming webhook url at slack. You can follow their guide on how
+To set up Slack, you need to configure an incoming Slack webhook URL. You can follow their guide on how
 to do that [here](https://api.slack.com/incoming-webhooks). If you want to include screenshots of the firing alerts
 in the Slack messages you have to configure either the [external image destination](#external-image-store) in Grafana,
 or a bot integration via Slack Apps. Follow Slack's guide to set up a bot integration and use the token provided
 (https://api.slack.com/bot-users), which starts with "xoxb".
 
 Setting | Description
 ---------- | -----------
-Url | Slack incoming webhook url.
+Url | Slack incoming webhook URL.
 Username | Set the username for the bot's message.
 Recipient | Allows you to override the Slack recipient.
 Icon emoji | Provide an emoji to use as the icon for the bot's message. Ex :smile:
 Icon URL | Provide a url to an image to use as the icon for the bot's message.
-Mention | make it possible to include a mention in the Slack notification sent by Grafana. Ex @here or @channel
+Mention Users | Mention one or more comma separated users in the Slack notification sent by Grafana. You have to refer to users via their Slack IDs (you can get a user's ID from their Slack profile).
```

Comment:
Could you maybe include an example url, I'm looking in the slack app but cannot find any id. Do you have to go to website/administration of slack workspace? Same in a few different places that makes me confused.

Reply:
@marefr I'm not sure what you mean by example URL. It's sort of difficult to explain more I think, without going really into detail with screenshots and all. Basically you have to f.ex. right click a username in chat and click their name to visit their profile, from where you can click the "More" button and then "Copy member ID".

I don't think it makes sense to be too detailed here about how to navigate Slack's UI, since it could change and then our documentation is stale.